src/WmPackageServiceProvider.php

```diff
@@ -25,6 +32,15 @@ public function boot()
     {
         parent::boot();
 
+        // Log failed jobs for both queue and horizon
```

Comment:
Prima fail test, poi quando capisci che evento è valido sposta la logica in un Listener